test/services/oauth2/hooks/index.test.js

```diff
@@ -0,0 +1,80 @@
+const { expect } = require('chai');
+const appPromise = require('../../../../src/app');
+const testObjects = require('../../helpers/testObjects')(appPromise);
+
+const { setIdToken } = require('../../../../src/services/oauth2/hooks/index');
+
+describe('oauth2 token test', () => {
+	let app;
+	let server;
+
+	before(async () => {
+		app = await appPromise;
+		server = app.listen(0);
+	});
+
+	after(() => {
+		server.close();
+	});
+
+	it('scope groups set', async () => {
```

Comment:
from reading the title of the test, its hard to infer what it does.
I always ask for three informations:
 - the unit under test (oauth2 token test)
 - the precondition, if any (scope groups set)
 - the behavior expected. for example, "should contain users team")
The last piece of information is missing in your tests. a nice way to achieve that, is to use nested describes
```
describe('oauth2 token test', () => {
  describe('when scope groups is set', () => {
    it.todo('should be defined')
    it.todo('should contain users team')
  });
});